### PR TITLE
Changes to use globals in the config yaml file as inputParamsJson

### PR DESCRIPTION
**Pull request by jayasimha-raghavan-unskript**
## Description

[EN-5057] 
Adding support to declare Global Variables used in Checks in the `unskript_config.yaml` file. 
This PR adds a sample template for reference.

### Testing
Please describe the tests that you ran to verify your changes. Please summarize what did you test and what needs to be tested e.g. deployed and tested helm chart locally. 

https://github.com/unskript/Awesome-CloudOps-Automation/assets/87547684/d3e70e23-f35d-4726-8bd4-6c57ac12febf

### Checklist:
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository. 

[EN-5058]: https://unskript.atlassian.net/browse/EN-5058?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[EN-5057]: https://unskript.atlassian.net/browse/EN-5057?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: unskript-ctl/db_utils.py [hunk 1]
```diff
@@ -173,6 +173,45 @@ def delete_pss_record(record_name: str, document_name: str) -> bool:
     return True
 
 
+def get_checks_by_uuid(check_uuid_list: list):
```
Comment (amit-chandak-unskript) [inline unskript-ctl/db_utils.py:176]: This is inefficient, we should query the DB to get this. Change the schema of the DB if required.

File: unskript-ctl/db_utils.py [hunk 2]
```diff
@@ -248,6 +287,8 @@ def get_creds_by_connector(connector_type: str):
     db.close()
 
     if not retval:
+        print("RETURNING RETVAL AS NONE, NONE")
```
Comment (amit-chandak-unskript) [inline unskript-ctl/db_utils.py:290]: Do we need this

File: unskript-ctl/unskript_config.yaml.template [hunk 3]
```diff
@@ -0,0 +1,13 @@
+globals:
+  namespace: "awesome-ops"
+  threshold: "string"
+  region: "us-west-2"
+  services: ["calendar", "audit"]
+
+
+suites:
```
Comment (amit-chandak-unskript) [inline unskript-ctl/unskript_config.yaml.template:8]: Remove the suites section.

File: unskript-ctl/unskript-client.py [hunk 4]
```diff
@@ -1449,6 +1523,8 @@ def stop_debug():
                         help='Run the given runbook FILENAME [-RUNBOOK_PARM1 VALUE1] etc..')
     parser.add_argument('-rc', '--run-checks', type=str,
                         help='Run all available checks [all | connector | failed]')
+    parser.add_argument('-rs', '--run-suites', type=str,
```
Comment (amit-chandak-unskript) [inline unskript-ctl/unskript-client.py:1526]: WE are not supporting this as of now, so suppress this.

File: unskript-ctl/unskript-client.py [hunk 5]
```diff
@@ -1449,6 +1523,8 @@ def stop_debug():
                         help='Run the given runbook FILENAME [-RUNBOOK_PARM1 VALUE1] etc..')
     parser.add_argument('-rc', '--run-checks', type=str,
```
Comment (amit-chandak-unskript) [inline unskript-ctl/unskript-client.py:1524]: run checks should support passing the parameters using the yaml file. Running suite gives the wrong impression.

File: unskript-ctl/unskript_config.yaml.template (hunk 3 above)
Comment (jayasimha-raghavan-unskript) [reply unskript-ctl/unskript_config.yaml.template:8]: Done. Removed

File: unskript-ctl/db_utils.py (hunk 1 above)
Comment (jayasimha-raghavan-unskript) [reply unskript-ctl/db_utils.py:176]: Dont get the comment. ZoDB does not have a search engine, we store list of dictionaries (code snippet list). The code queries the ZoDB and fetch the checks that match the check UUIDs that were sent as a list.

File: unskript-ctl/db_utils.py (hunk 2 above)
Comment (jayasimha-raghavan-unskript) [reply unskript-ctl/db_utils.py:290]: No We dont need this.

File: unskript-ctl/unskript-client.py (hunk 4 above)
Comment (jayasimha-raghavan-unskript) [reply unskript-ctl/unskript-client.py:1526]: sure. Commenting the code out.

File: unskript-ctl/unskript_config.yaml.template (hunk 3 above)
Comment (jayasimha-raghavan-unskript) [reply unskript-ctl/unskript_config.yaml.template:8]: Done

**Review by amit-chandak-unskript — APPROVED** [review]

**Pull request merged**